src/v2/Apps/Fairs/Components/FairsFairRow.tsx

```diff
@@ -0,0 +1,107 @@
+import { Box, BoxProps, ChevronIcon, color, Image, Text } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { FairsFairRow_fair } from "v2/__generated__/FairsFairRow_fair.graphql"
+
+const Container = styled(Box)`
+  transition: background-color 250ms;
+
+  &:hover {
+    background-color: ${color("black5")};
+  }
+`
+
+interface FairsFairRowProps extends BoxProps {
+  fair: FairsFairRow_fair
+}
+
+const FairsFairRow: React.FC<FairsFairRowProps> = ({ fair, ...rest }) => {
+  const icon = fair?.profile?.icon?.resized
+
+  return (
+    <RouterLink
+      to={fair.href}
+      style={{ display: "block", textDecoration: "none" }}
+    >
+      <Container
+        display="flex"
+        alignItems="center"
+        borderBottom="1px solid"
+        borderColor="black10"
+        py={2}
+        px={[0, 2]}
+        {...rest}
+      >
+        <Box
+          display="flex"
+          flex="1"
+          alignItems={["flex-start", "center"]}
+          flexDirection={["column", "row"]}
+        >
```

Comment:
(nonblocking) No longer use `<Flex>`? IMO it reads a bit better at a glance, and is what most people use.

Reply:
I personally find it more confusing; having to switch between two components for a single display option.

Reply:
This way is totally cool, but its def unconventional across our UI code in Force / Eigen.